### PR TITLE
The constant is called SHORTINIT, not SHORT_INIT

### PR DESCRIPTION
**Pull request by mwidmann**

### CODE REVIEW

**Comment by mwidmann** [comment]
Great talk at #wpeu btw. 👍

File: a-constants-reference.md [hunk 1]
```diff
@@ -17,7 +17,7 @@
 	_Default Value_: `SITECOOKIEPATH . 'wp-admin'`
 
 	_Description_: Admin backend path to store the cookie under.
-
+Fshort_
```
Comment (schlessera) [inline a-constants-reference.md]: This looks like a search artifact...

**Review by schlessera — CHANGES_REQUESTED** [review]
There's some bit of random text included in the PR, probably a mistyped search request.

Please remove this artifact, so that I can merge the PR.

**Comment by mwidmann** [comment]
I removed the additional text, no idea how that came to be.

**Review by schlessera — APPROVED** [review]

**Comment by schlessera** [comment]
Thanks for the PR, @mwidmann !

**Pull request merged**